.openshift-ci/crawler/crawl.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# push changes
+BRANCH="$(echo "$JOB_SPEC" | jq -r '.extra_refs[0].base_ref')"
+
+# Assume we need to run in staging mode unconditionally for testing purposes.
+
+source .openshift-ci/env.sh
+source .openshift-ci/google-cloud-sdk/install.sh
+source .openshift-ci/google-cloud-sdk/init.sh
+source .openshift-ci/crawler/setup-staging.sh
+
+# temporary clean-up package list to reduce crawling time
+shopt -s extglob
+rm kernel-package-lists/!(centos.txt|centos-uncrawled.txt|rhel.txt|rhel-uncrawled.txt|reformat.yml)
+cat <<EOT > kernel-package-lists/reformat.yml
+- name: centos
+  description: CentOS kernels
+  type: redhat
+  file: centos.txt
+  reformat: single
+
+- name: rhel
+  description: RHEL
+  type: redhat
+  file: rhel.txt
+  reformat: single
+
+EOT
```

Comment:
Why are you having a script change reformat.yml, when you could just make the changes? Perhaps you want to keep reformat.yml as it is for CircleCI and local use, but want it to be different when running in Openshift CI. Perhaps consider creating another file to be used by Openshift CI.